pkg/scaling/executor/scale_jobs.go

```diff
@@ -41,6 +41,13 @@ const (
 func (e *scaleExecutor) RequestJobScale(ctx context.Context, scaledJob *kedav1alpha1.ScaledJob, isActive bool, scaleTo int64, maxScale int64) {
 	logger := e.logger.WithValues("scaledJob.Name", scaledJob.Name, "scaledJob.Namespace", scaledJob.Namespace)
 
+	isPaused := e.Paused(ctx, logger, scaledJob)
```

Comment:
A better place to have this assessment might be in `pkg/scaling/scale_handler.go` however the original MR had the referenced method in this file, so I've put here for now.

Reply:
I think that we should manage this at controller level. If the job is paused we can avoid the work until there just actuating in the controller. WDYT @zroubalik ?